imports/plugins/included/payments-example/server/no-meteor/i18n/en.json

```diff
@@ -3,7 +3,7 @@
   "i18n": "en",
   "ns": "example-paymentmethod",
   "translation": {
-    "reaction-payments": {
+    "example-paymentmethod": {
```

Comment:
Same comment for all translation files in this package: Should this be `reaction-payments-example` or just `payments-example` to match the package name?

Reply:
Discussion in the other comment.